src/content/blog/2024-09-29-fighting-the-invisible-Battle-my-story-of-health-struggles.mdx

```diff
@@ -0,0 +1,170 @@
+coverSVG: ../images/svg/undraw/drew-beamer-xU5Mqq0Chck-unsplash.svg
+socialImage: ../images/health/wellness.png
+draft: false
+tags: ['health','family','community']
+categories: ['community']
+publishDate: 2024-10-08T20:48:29.436Z
+author: Santosh Yadav
+---
+
+import doubt from '../../images/health/doubt.jpg';
+import myself from '../../images/health/healthy.jpg';
+import wellness from '../../images/health/wellness.png';
+import speaking from '../../images/undraw/public-speaking.jpeg';
+import excuse from '../../images/health/excuse.jpg';
+
+import BlogImage from '../../components/blogimage.astro';
+
+Hey friends, if you are here, I wish you a healthy life and hope you are doing great physically and mentally. And please take care of your health. I learned it the hard way. Though I always preach about work-life balance, I ended up screwing up my health. 
+
+I will share my journey, how I fought an invisible battle, and where I am currently.
+
+# What happened 
+
+In April 2024, my wife and I went for a full-body checkup, as we had hardly done it since we moved to Germany. We got our reports after a week, and my doctor gave me the bad news, I had symptoms of diabetes, and my blood sugar was 270 in the report. She was surprised at how I could do my day-to-day work as I should have mainly been sleeping; she was not wrong; that was what I was doing, and my doctor mentioned she could write me a note to take some days off.
+
+## Struggle with health
+
+In the past three years, I have doubted myself thousands of times. If I was a good engineer, one thing that would have helped me succeed over the years would be my confidence that I could do anything. 
+
+### I started to question myself: how did I end up here? 
+
+It was all stress when Peter Parker stopped believing in himself in Spider-Man. I am not saying I am a superhero, but it was the same: everything was due to stress.
+
+What happened 3 years ago, in Jan 2022, was I moved to Germany, and I was away from my family for 10 months, which caused me mental stress. With no support in a new country, it started impacting my health.
+
+I have been under stress before; I struggled financially for almost my entire life and went through a phase where me and my wife were fighting every day to save our daughter and praying to god that she would come back home soon. 
+
+But I always had my wife, daughter, and some amazing friends by my side, and we survived that phase. But not having my family and friends after moving to a new country was painful.
+
+> What happened 3 years ago as I moved to Germany, and I was away from my family for 10 months, which caused me mental stress. With no support in a new country, it started impacting my health.
+
+### Impact on work
+
+After moving to Germany, I started underperforming at my first job, and I saw no future at the organization. I was lucky to join Celonis in Aug 2022, where my co-workers and employer gave me the support I wanted. 
```

Comment:
The article jumps between different time periods. It may help maintain a chronological order to avoid confusion, especially when reflecting on the time before and after moving to Germany and when discussing your diagnosis.

Reply:
Adjusted this a bit